FWCore/Framework/test/esproducer_t.cppunit.cc

```diff
@@ -85,13 +85,13 @@ class ShareProducer : public ESProducer {
       ptr_->value_ = 0;
       setWhatProduced(this);
    }
-   boost::shared_ptr<DummyData> produce(const DummyRecord& /*iRecord*/) {
+   std::shared_ptr<DummyData> produce(const DummyRecord& /*iRecord*/) {
       ++ptr_->value_;
       std::cout <<"produce called "<<ptr_->value_<<std::endl;
       return ptr_;
    }
 private:
-   boost::shared_ptr<DummyData> ptr_;
+   std::shared_ptr<DummyData> ptr_;
 };
 
```

Comment:
NOTE: This would be the file where I would test to see if returning an `std::unique_ptr` from `produce` would work.